package.json

```diff
@@ -42,19 +46,21 @@
   },
   "dependencies": {
     "@dsmjs/components": "2.2.0",
-    "gatsby": "1.9.272",
-    "gatsby-link": "1.6.46",
-    "gatsby-plugin-glamor": "2.0.5",
+    "gatsby": "^2.0.0",
```

Comment:
i'm surprised these are being `save-exact`ed. i usually go that route for the primary app or site. this is a good reminder for me to get that configured separately

Reply:
You mean _not_ being save exacted, or the fact that they were (existing) save exacted?

I don't care either way, so I'll go with whatever!

Reply:
sorry, yeah. the intent is that they are exact, which should happen automatically based on the [`.npmrc`](https://github.com/dsmjs/site/blob/master/.npmrc) when installed using the cli. was this a yarn thing? (i dont know how much yarn impacts the `package.json`)